frontend/components/component/editor.tsx

```diff
@@ -146,12 +148,35 @@ export default function Editor() {
   const handleDownload = () => {
     const link = document.createElement("a");
     link.href = localStorage.getItem("tiffUrl")!;
-    link.download = "georeferenced.tiff";
+    link.download = `${projectName}.tiff`;
```

Comment:
Make sure to change spaces to underscore in the string.

Reply:
Pushed an update; added a new var for normalized name, on update spaces are replaced with underscores, slashes with dashes, and A-Z, a-z, 0-9, - and _ are allowed. The user still sees all their characters, this only affects the file name of downloaded file and name stored in backend.